front-end/js/jump.js

```diff
@@ -362,6 +392,171 @@ function registerJumps(viewId, svg, layerId) {
         // make cursor a hand when hovering over this shape
         d3.select(this).style("cursor", "zoom-in");
 
+        // disable context menu on kyrix objects since we use right clicks for update
```

Comment:
I think we should add an option to disable updates -- adding a layer-level property in the compiler would work.